src/browser-frontend/views/browser/chrome.jsx

```diff
@@ -14,13 +14,15 @@ import React, { PureComponent } from 'react';
 
 import WindowControls from './chrome/decorations/window-controls';
 import TabBar from './chrome/tabbar';
+import AppMenu from './chrome/app-menu';
 
 export default class Chrome extends PureComponent {
   render() {
     return (
       <div>
         <WindowControls />
         <TabBar />
+        <AppMenu />
```

Comment:
I like this because it fits in with the rest of the frontend.  But if at some point we pulled the menu out into another process (at least conceptually - up to the platform/runner) this wouldn't work and we'd instead need some kind of messaging.  I don't think we should worry about that now though since I don't think we can get the desired menu layout using the Menu API from electon.  So let's go with this since it allows arbitrary html and is relatively easy to work on for prototypes.